service/src/test/java/bio/terra/tanagra/service/accesscontrol/impl/MockVumcAdminAccessControl.java

```diff
@@ -24,7 +24,8 @@ protected boolean apiIsAuthorizedUser(String userEmail) {
   }
 
   @Override
-  protected ResourceList apiListAuthorizedResources(String userEmail, ResourceType resourceType) {
+  protected ResourceList apiListAuthorizedResources(
+      String userEmail, ResourceType resourceType, String googleGroup) {
```

Comment:
minor: change param name to userAccessGroup or accessGroup

Reply:
missed that. Will do the change before merge.  Thanks.